bpf/lib/maps.h

```diff
@@ -213,7 +213,7 @@ struct {
 	__type(key, struct egress_gw_policy_key);
 	__type(value, struct egress_gw_policy_entry);
 	__uint(pinning, LIBBPF_PIN_BY_NAME);
-	__uint(max_entries, EGRESS_POLICY_MAP_SIZE);
+	__uint(max_entries, 16384);
```

Comment:
This is to keep the BPF unit tests happy.

Reply:
So, value must be hardcoded and constant is not allowed?

Reply:
I removed the define for `EGRESS_POLICY_MAP_SIZE` so that I can get rid of the entry in `node_config.h` which is written out at runtime. It's difficult to plumb the size requested via config / command line into `node_config.h` without relying on `option.Config` global which I wanted to avoid.

So: we could keep the `define` but move it to `maps.h`. That runs the risk of some C code relying on that define with the assumption that it reflects the configured value. So I opted to remove the define instead.

Reply:
Thanks!